### PR TITLE
Docs textwrap, typo fixes, add links

### PR DESCRIPTION
**Pull request by hoodmane**
I textwrapped a few files, added a few API reference links in faq.md and loading-packages.md, and normalized capitalization again (PYODIDE, pyodide ==> Pyodide).

### CODE REVIEW

File: docs/project/governance.md [hunk 1]
```diff
@@ -1,57 +1,58 @@
-# pyodide governance and decision-making
+(governance)=
+# Pyodide Governance and Decision-making
```
Comment (hoodmane) [inline docs/project/governance.md:2]: Capitalization in title

File: docs/project/governance.md [hunk 2]
```diff
@@ -1,57 +1,58 @@
...
 
 ### Contributors
 
 Contributors are community members who contribute in concrete ways to the
-project. Anyone can become a contributor, and contributions can take many
-forms, for instance, answering user questions – not only code – as detailed in
-the {ref}`how_to_contibute`
+project. Anyone can become a contributor, and contributions can take many forms,
+for instance, answering user questions – not only code – as detailed in
+{any}`how_to_contribute`.
```
Comment (hoodmane) [inline docs/project/governance.md:21]: Broken link ("contibute" was missing the r)

File: docs/usage/faq.md [hunk 3]
```diff
@@ -16,12 +16,15 @@ In both cases, files need to be served with a web server and cannot be loaded fr
...
+For security reasons Javascript in the browser is not allowed to load local data
+files. You need to serve them with a web-browser. Recently there is a
+[Native File System API](https://wicg.github.io/file-system-access/) supported in Chrome
+but not in Firefox.
+[There is a discussion about implementing it for Firefox here.](https://github.com/mozilla/standards-positions/issues/154)
 
 
-## How can I change the behavior of `runPython` and `runPythonAsync`?
-The definitions of `runPython` and `runPythonAsync` are very simple:
+## How can I change the behavior of {any}`runPython <pyodide.runPython>` and {any}`runPythonAsync <pyodide.runPythonAsync>`?
```
Comment (hoodmane) [inline docs/usage/faq.md:26]: Added links in faq

File: docs/usage/loading-packages.md [hunk 4]
```diff
@@ -4,11 +4,11 @@
 Only the Python standard library is available after importing Pyodide.
 To use other packages, you’ll need to load them using either:
  - {any}`pyodide.loadPackage` for packages built with Pyodide, or 
- - `micropip.install` for pure Python packages with wheels available on PyPi or
+ - {any}`micropip.install` for pure Python packages with wheels available on PyPi or
```
Comment (hoodmane) [inline docs/usage/loading-packages.md:7]: Added links in loading-packages

**Review by rth — APPROVED** [review]
Thanks @hoodmane !

Feel free to merge.

**Pull request merged**